.github/workflows/releaser.yml

```diff
@@ -0,0 +1,68 @@
+name: Releaser
+on:
+  push:
+    branches-ignore:
+      - '**'
+    tags:
+      - 'v*.*.*'
+      - 'v*.*.*-*'
+
+jobs:
+
+  build_linux_amd64:
+    runs-on: ubuntu-latest
+    steps:
+         - uses: actions/checkout@v2
+         - name: build avail explorer via yarn
+           shell: bash
+           run: |
+            yarn install
+            yarn run build:www
+            pushd /home/runner/work/avail-apps/avail-apps/packages/apps/
+            tar czf avail-apps-linux-amd64.tar.gz build/
```

Comment:
This is just a static website, so I think you can just name it like... `avail-explorer.tar.gz`